### DISCUSSION TITLE
PyQt/PySide support

### DISCUSSION THREAD

**Discussion by designingSparks** [indygreg/PyOxidizer#427] [Q&A]
Is this planned at all? IMO, this would be a killer feature.

**Comment by dae** [comment]
PyQt works for me - just copy it next to the binary, and then set sys.path so it can be found.

**Reply by designingSparks** [reply]
Can you please elaborate on this? Do you mean you copy all DLLs next to the main binary? PyQt has many DLLs.

**Comment by dae** [comment]
The entire PyQt5/6 folder from your site packages or a wheel. You could optionally strip parts out, but a bunch of dlls depend on each other, so you'd need to experiment to see what is essential and what can be culled for your app.
